library/public/utils.h

```diff
@@ -68,6 +68,30 @@ class F3D_EXPORT utils
   [[nodiscard]] static std::filesystem::path collapsePath(
     const std::filesystem::path& path, const std::filesystem::path& baseDirectory = {});
 
+  /**
+   * Returns true if the glob contains a globstar (`**`).
+   */
+  [[nodiscard]] static bool globContainsGlobstar(std::string_view glob);
```

Comment:
from libf3d pov the globstar detection could be put into globToRegex in order to avoid exposing another method in utils.cxx.

But maybe the API you suggest here is better and would be more expected by libf3d users, wdyt ?

Reply:
I agree with @snoyer here :)